apps/vessel-history/features/profile/components/Info.module.css

```diff
@@ -22,9 +22,45 @@ div.identifierField {
   color: var(--color-primary-blue);
 }
 
```

Comment:
The responsive styles to display the forced labour properly on tablets are missing and the design breaks when viewing it on the galaxy tab active 2.
<img width="600" alt="image" src="https://user-images.githubusercontent.com/2205831/183071720-d3bb227e-8f19-41b1-b91a-562ed95d7a4d.png">